rj_gameplay/stp/formations/x_formation.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
+
+import numpy as np
+
+import stp
+from stp.utils.constants import RobotConstants
+
+
+class X_formation(stp.formations.Formations):
```

Comment:
```suggestion
class XFormation(stp.formations.Formations):
```

same nit as above with Diamond_formation vs DiamondFormation